Fixed information about UMSG_DROPREQ (#87)
Haivision/srt-rfc: draft-sharabayko-mops-srt.md

A Message Drop Request control packet is sent by the sender to the receiver
when it requests the retransmission of an unacknowledged packet (all or part
of a message) which is not present in the sender's buffer. This may happen, for
example, when a TTL parameter (passed in the sending function) triggers a

> timeout for retransmitting lost packets being parts of the message, causing

Reviewer: ```suggestion
timeout for retransmitting lost packets which constitute parts of a message, causing
```

> this time, so the packet was removed from the sender buffer.

Reviewer 2: - We don't have "file/message" mode defined in RFC.
- `srt_sendmsg` is library specific API function, not defined in the RFC. 

Consider generalizing this. Maybe, @stevomatthews , you could help here?

> of a message) which is not present in the sender's buffer. This may happen
> in the file/message mode when the message TTL parameter in `srt_sendmsg`
> was a defined timeout and the message was given up retransmission after
> this time, so the packet was removed from the sender buffer.

Reviewer: ```suggestion
of a message) which is not present in the sender's buffer. This may happen, for 
example, when a TTL parameter triggers a timeout, causing a message/packet 
to be removed from the sender's buffer.
```

Reviewer 3: > causing a message/packet to be removed 

I think the method of dropping a message or packet will be different. I guess dropping multiple packets repeatedly is the way to drop a single message.

IMHO, If the deletion of "message" is needed here, it should also be presented how to achieve it. Or, simply, it can be "packet".

Author: Ok, I made it a little bit more precise; @stevomatthews please re-review.
